config/jobs/kubernetes/kops/build-pipeline.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+template = """
+# Verify the latest-ci version from the {{branch}} branch of kops
+# Runs a small subset of the e2e tests.
+# Publishes the version to latest-ci-updown-green on success.
+- interval: 60m
+  name: e2e-kops-pipeline-updown-{{branch}}
+  decorate: true
+  decoration_config:
+    timeout: 45m
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  spec:
+    containers:
+    - image: {{e2e_image}}
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --up
+      - --test
+      - --down
+      - --dump=$(ARTIFACTS)
+      - --timeout=45m
+      - --gcp-service-account=$(E2E_GOOGLE_APPLICATION_CREDENTIALS)
+      # kops-specific test args
+      - --deployment=kops
+      - --provider=aws
+      - --cluster=e2e-kops-pipeline-updown-{{branch}}.test-cncf-aws.k8s.io
+      - --kops-ssh-user=admin
```

Comment:
This is going to be branch specific, with newer branches using "ubuntu"

Reply:
Well spotted - will fix!